### ISSUE TITLE
does elastic bean stalk  work with Django==3.0.5

### ISSUE THREAD

**Issue by arunjose696** [aws/aws-elastic-beanstalk-cli-setup#77]
I am following the below tutorial

https://docs.aws.amazon.com/elasticbeanstalk/latest/dg/create-deploy-python-django.html#python-django-deploy

I am able to deploy django app for version django==2.1.1 as in the doc. However when I try to deploy django application with django version(Django==3.0.5)  I am gettig an internal server error. 
I could see  in the document it was mentioned "The latest compatible version is Django 2.1" but could read in other github articles that latest versions of django like django 3 are supported by elastic bean stalk. Could anyone please help me with steps to deploy a an application with Django==3.0.5 using elastic beanstalk

**Comment by arunjose696** [comment]
Hi Everyone,

If anybody else is facing this issue, I found a possible implementation in the below blog:

[https://igonato.com/django-3-celery-aws-elastic-beanstalk/]

I was not able to try this out as I am busy with some work. Would appreciate if anybody could try ou and vouch for it

**Comment by punisher21maximum** [comment]
You can not use django 3.0 with elastic bean stalk

**Comment by arunjose696** [comment]
hey @punisher21maximum ,

have you tried out the blog post??

**Comment by weynelucas** [comment]
> Hi Everyone,
> 
> If anybody else is facing this issue, I found a possible implementation in the below blog:
> 
> [django-3_ AWS](https://igonato.com/django-3-celery-aws-elastic-beanstalk/)
> 
> I was not able to try this out as I am busy with some work. Would appreciate if anybody could try ou and vouch for it

The link is off

**Comment by jasonhoward64** [comment]
> 
> 
> You can not use django 3.0 with elastic bean stalk

This is categorically wrong.

**Comment by chiragchamoli** [comment]
> > You can not use django 3.0 with elastic bean stalk
> 
> This is categorically wrong.

No, You can't I have tried this and they don't support anything after Django 2.2.

**Comment by jasonhoward64** [comment]
> > > You can not use django 3.0 with elastic bean stalk
> > 
> > 
> > This is categorically wrong.
> 
> No, You can't I have tried this and they don't support anything after Django 2.2.

I spoke to someone at aws via phone. Their documentation is out of date. He told me the same thing (that django 3.0 doesn't work on EB). He went quiet went I showed him a deployment on EB that was django 3.0+ and then admitted that their docs can be out for date.

**Comment by chiragchamoli** [comment]
> > > > You can not use django 3.0 with elastic bean stalk
> > > 
> > > 
> > > This is categorically wrong.
> > 
> > 
> > No, You can't I have tried this and they don't support anything after Django 2.2.
> 
> I spoke to someone at aws via phone. Their documentation is out of date. He told me the same thing (that django 3.0 doesn't work on EB). He went quiet went I showed him a deployment on EB that was django 3.0+ and then admitted that their docs can be out of date.
> 
> In summary, I'm currently running django 3+ on aws eb with 0 issues.

Will try man.

**Comment by theamanbhargava** [comment]
It does not work if you rely on any GIS features: https://github.com/aws/aws-elastic-beanstalk-cli-setup/issues/77

**Comment by ScottHull** [comment]
I have not been able to get anything past Django 2.2 running on EB.  Even then, getting GIS features running on EB is a nightmare.  For some reason the open issues around full Django support have been closed for no reason.

**Comment by jasonhoward64** [comment]
Not sure what module you're using for GIS, but it must be the reason for
the failure.

Vanilla django 3.0 works perfect with eb / aws.

What error are you getting?

**Comment by 3rickDomQ** [comment]
> > > > You can not use django 3.0 with elastic bean stalk
> > > 
> > > 
> > > This is categorically wrong.
> > 
> > 
> > No, You can't I have tried this and they don't support anything after Django 2.2.
> 
> I spoke to someone at aws via phone. Their documentation is out of date. He told me the same thing (that django 3.0 doesn't work on EB). He went quiet went I showed him a deployment on EB that was django 3.0+ and then admitted that their docs can be out of date.
> 
> In summary, I'm currently running django 3+ on aws eb with 0 issues.

Could u post a light guide on how to do it? Also if u have any insights about GIS deploy it would all be greatly appreciated, thanks!

**Comment by jasonhoward64** [comment]
Not sure what GIS. My suggestion would be to try deploying a django project
without GIS. Just something simple. Get it running and then start adding
and see what breaks. Your dependencies might work locally but the might not
work with the dependencies that EB installs. Maybe it doesn't like your
versions for example.

If you have troubles deploying a vanilla django project to EB, send me a
note and I'll help.

**Comment by siddharth-kt** [comment]
@ScottHull @3rickDomQ 
How did you manage to deploy Django + GIS libs on ElasticBeanstalk ?
